go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/Masterminds/semver/v3 v3.2.1
 	github.com/btcsuite/btcd/btcec/v2 v2.3.2
 	github.com/btcsuite/btcd/btcutil v1.1.5
-	github.com/cometbft/cometbft-db v0.10.0
+	github.com/cometbft/cometbft-db v0.11.0
```

Comment:
Do we need any not in the `CHANGELOG` that we are upgrading this ? I think this is just the compaction changes for the DB and not breaking as far as I understand

Reply:
> I think this is just the compaction changes for the DB and not breaking as far as I understand

correct